### DISCUSSION TITLE
Set up an SFTP Server in Windows

### DISCUSSION THREAD

**Discussion by Samson1974** [iterate-ch/cyberduck#15653] [Q&A] [answered]
Hey all,

Yeah, so I am wondering. How would one go about setting up an SFTP server in Cyberduck. I kind of understand how to connect, but what if I want to, actually, host some files, so others can come to the desktop PC and download or upload. Looking forward to some answers. Sorry if this has already been answered, but I can not find any information, anywhere, on the internet.

thanks in advance

**Comment by tatze96** [comment, marked as answer]
Cyberduck is intended to access existing servers and cloud storages for file transfer and management. It can't create and provide servers.

If you already have access to an SFTP server, reach out to your server administrator to get the information on which hostname, credentials, and, if necessary, the public key need to be used to connect to the server successfully.

Note that connections made through Cyberduck can't be shared within the local network.

**Reply by Samson1974** [reply]
Oh okay, I appreciate that. Have to figure out how to set up a server, so that, supposedly, encrypted transfers can take place, between friends. Just need to generate and hand-out certificates and password-protect the certificate, somehow, for additional security, on the server-side. I'll probably use, SFTP, as a Network transfer protocol.
